packages/endo/bin/endo.js

```diff
@@ -2,6 +2,7 @@
 /* global process */
 (async () => {
   const fs = await import('fs');
+  const crypto = await import('crypto');
   const { main } = await import('../src/cli.js');
-  main(process, { fs: fs.promises });
+  main(process, { fs, crypto });
```

Comment:
How does this change from `fs.promises` to just `fs` work?

Reply:
This comes with the shift to using `makeNodePowers(fs, crypto)` that happens elsewhere. The new stuff does the unpacking from the bare module namespace.